index.html

```diff
@@ -343,50 +352,54 @@ <h3>
 
 <p><h2>Status Codes:</h2></p>
```

Comment:
Every title in <h2> tag make in one style : or with ':', or with '.', or without punctuation marks at the end